lib/teracy-dev-core/config/virtualbox_provider.rb

```diff
@@ -12,7 +12,15 @@ def configure_node(settings, config)
           @logger.debug("provider_settings: #{provider_settings}")
           case provider_settings['type']
           when "virtualbox"
-            # TODO: check require_version
+            current_version = `VBoxManage --version`
+
+            if !TeracyDev::Util.require_version_valid?(
+              current_version, provider_settings['require_version'])
+              @logger.error("Your current virtualbox (#{current_version}) is not up to date.\
+                The required version is #{provider_settings['require_version']}.
+                Please upgrade it to the required version then run `VBoxManage --version` to check.".squeeze)
+            end
```

Comment:
add `abort` at the end of the function to stop the program in case error happens.